test/utils/Votes.t.sol

```diff
@@ -0,0 +1,200 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import {Votes} from "src/utils/Votes.sol";
+import {Test, console2} from "@std/Test.sol";
+
+contract VotesTest is Votes {
+    mapping(address => uint256) public balances;
+
+    function mint(address account, uint256 amount) external {
+        balances[account] += amount;
+        transferVotingUnits(address(0), account, amount);
+    }
+
+    function transfer(address recipient, uint256 amount) external {
+        address sender = msg.sender;
+        balances[sender] -= amount;
+        balances[recipient] += amount;
+        transferVotingUnits(sender, recipient, amount);
+    }
+
+    function balanceOf(address owner) external view override returns (uint256 balance) {
+        return balances[owner];
+    }
+
+    function name() external pure override returns (string memory) {
+        return "VotesTest";
+    }
+
+    function version() external pure override returns (string memory) {
+        return "1";
+    }
+}
+
+abstract contract VotesTestHelper is Test {
+    address public deployer = address(0x1);
+    address public owner = address(0x2);
+    address public recipient = address(0x3);
+
+    VotesTest public votes;
+
+    function setUp() public {
+        vm.prank(owner);
+        votes = new VotesTest();
+    }
+}
+
+contract BaselineTest is VotesTestHelper {
+    function testGetVotes() public {
+        votes.mint(recipient, 100);
+        assertEq(votes.getVotes(recipient), 0);
+
+        vm.prank(recipient);
+        votes.delegate(recipient);
+        assertEq(votes.getVotes(recipient), 100);
+    }
+
+    function testGetPastVotes() public {
+        vm.warp(42);
+        votes.mint(recipient, 100);
+        assertEq(votes.getVotes(recipient), 0);
+
+        vm.prank(recipient);
+        votes.delegate(recipient);
+        assertEq(votes.getPastVotes(recipient, 1), 0);
+        assertEq(votes.getPastVotes(recipient, 42), 100);
+    }
```

Comment:
We don't have to do this here, but we should expand on this `BaselineTest` contract so that we cover all the public functions and exercise their basic use cases.